core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -466,6 +466,12 @@ public final void runInContext() {
             closed = true;
             cancelListenersShouldBeRemoved = true;
             observer.onClose(savedStatus, savedTrailers);
+          } catch (RuntimeException e) {
```

Comment:
yes, I had intended to fill them with log statements, but didn't know if they should be caught by the executor